code-of-conduct.md

```diff
@@ -0,0 +1,39 @@
+## Introduction
+
+Through participation in Moonlight Remote Work Encyclopedia, you will be interacting with other members of the Moonlight community, including both developers and clients. One of the core components of this is the Moonlight Community Slack Channel. These Community Guidelines and related procedures also apply to unacceptable behavior occurring outside the scope of community activities, in all community venues— online and in-person— as well as in all one-on-one communications, and anywhere such behavior has the potential to adversely affect the safety and well-being of community members.
+
+Your job engagements through Moonlight are covered by specific confidentiality requirements in the [Terms of Service](/terms). However, please treat all communications through Moonlight, including its Slack channel, as off-the-record and confidential information.
+
+Diversity and inclusion make our community strong. We encourage participation from the most varied and diverse backgrounds possible and want to be very clear about where we stand. Our goal is to maintain a safe, helpful and friendly community for everyone, regardless of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other defining characteristic.
+
+## Expected Behavior
+
+* Be welcoming.
+* Be kind.
+* Look out for each other.
+* Treat messages, notifications, and emails shared within the Moonlight community as confidential, and do not share them externally.
+
+## Unacceptable Behavior
+
+* Conduct or speech which might be considered sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory or offensive in nature.
+* Do not use unwelcome, suggestive, derogatory or inappropriate nicknames or terms.
+* Do not show disrespect towards others. (Jokes, innuendo, dismissive attitudes.)
+* Intimidation or harassment (online or in-person). Please read the Citizen Code of Conduct for how we interpret harassment.
+* Disrespect towards differences of opinion.
+* Inappropriate attention or contact. Be aware of how your actions affect others. If it makes someone uncomfortable, stop.
+* Not understanding the differences between constructive criticism and disparagement.
+* Sustained disruptions.
+* Violence, threats of violence or violent language.
+* Marketing or self-promotion
+* Posting jobs from websites other than Moonlight
+* Posting links without commentary, context, or active participation
+
+## Enforcement
+
+Understand that speech and actions have consequences, and unacceptable behavior will not be tolerated. 
+
+If you are the subject of, or witness to any violations of these Community Guidelines, please contact us by emailing [team@moonlightwork.com](mailto:team@moonlightwork.com).
+
+If violations occur, Moonlight will take any action it deems appropriate for the infraction, up to and including account termination.
```

Comment:
Not sure account termination means much in this case but it could still apply to our members.